spec/models/blacklight/solr/search_builder_spec.rb

```diff
@@ -240,6 +240,14 @@
     describe "solr parameters for a field search from config (subject)" do
       let(:user_params) { subject_search_params }
 
+      before do
+        # pre solr-7.2-style configuration with local params
```

Comment:
I'm not sure what this comment is telling me.  Is it "You can set..."?

Reply:
The tests in this block expect the solr local params, which don't work at all in Solr 7.2, so we have to set that configuration in this unit test to test the feature.